terraform/031-email-service/task-definition-api.json

```diff
@@ -25,10 +25,6 @@
         "name": "APP_ID",
         "value": "${appconfig_app_id}"
       },
-      {
```

Comment:
Why remove this?

Reply:
It's duplicated. You'll see the same entry on (old) lines 52-55.